wallet/src/de/schildbach/wallet/ui/main/WalletActivityExt.kt

```diff
@@ -179,13 +199,14 @@ object WalletActivityExt {
     }
 
     private fun WalletActivity.showLowStorageAlertDialog() {
-        val storageManagerIntent = Intent(
-            if (Build.VERSION.SDK_INT >= Build.VERSION_CODES.O) {
-                StorageManager.ACTION_MANAGE_STORAGE
-            } else {
-                Settings.ACTION_MANAGE_APPLICATIONS_SETTINGS
-            }
-        )
+        val storageManagerIntent =
+            Intent(
+                if (Build.VERSION.SDK_INT >= Build.VERSION_CODES.O) {
+                    StorageManager.ACTION_MANAGE_STORAGE
+                } else {
+                    Settings.ACTION_MANAGE_APPLICATIONS_SETTINGS
+                }
+            )
```

Comment:
Is there a way to get the `ktlint` plugin for Android Studio to use our rules in this project?


This was another case where I was using the new ktlint plugin to reformat a file on save, which would actually change a lot in `wallet` and in other modules may apply rules that we don't use.

Reply:
I think if you set `.editorconfig` as the baseline file in ktlint plugin settings, it should pickup our rules.
I removed ktlint plugin from the studio because after some update it broke and started to complain about everything.